integration_test/create_account_test.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:provenance_wallet/common/widgets/pw_onboarding_screen.dart';
+import 'package:provenance_wallet/main.dart' as app;
+import 'package:provenance_wallet/screens/account_name_screen.dart';
+import 'package:provenance_wallet/screens/account_setup_confirmation_screen.dart';
+import 'package:provenance_wallet/screens/account_type_screen.dart';
+import 'package:provenance_wallet/screens/backup_complete_screen.dart';
+import 'package:provenance_wallet/screens/create_passphrase_screen.dart';
+import 'package:provenance_wallet/screens/enable_face_id_screen.dart';
+import 'package:provenance_wallet/screens/home/dashboard/dashboard.dart';
+import 'package:provenance_wallet/screens/landing/landing_screen.dart';
+import 'package:provenance_wallet/screens/pin/pin_pad.dart';
+import 'package:provenance_wallet/screens/recovery_words/recovery_words_screen.dart';
+import 'package:provenance_wallet/screens/recovery_words_confirm/recovery_words_confirm_screen.dart';
+import 'package:provenance_wallet/screens/recovery_words_confirm/word_button.dart';
+import 'package:provenance_wallet/screens/recovery_words_confirm/word_selector.dart';
+
+import 'util/key_extension.dart';
+import 'util/widget_tester_extension.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets(
+    'Starting from nothing, should create basic account.',
+    (tester) async {
+      app.main([]);
+
+      final testData = await tester.loadTestData();
+
+      await tester.pumpAndSettle(Duration(seconds: 1));
```

Comment:
If we await `mainCompleter` can we remove the duration?

Reply:
Yes. I'll get that.